docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -438,6 +440,53 @@
       your Integrated Spark Master node's IP address.</para>
 
       <programlisting>https://192.168.56.101:8080</programlisting>
+
+      <sect2 id="Addl_Spark_Config">
+        <title>Integrated Spark Cluster Configuration Options</title>
+
+        <para>To customize your Integrated Spark cluster environment utilize
+        the provided <emphasis role="bold">spark-env.sh</emphasis> script.
+        Using the spark-env.sh script you can specify additional configuration
+        options for your Integrated Spark cluster.</para>
+
+        <programlisting>/etc/HPCCSystems/externals/spark/spark-env.sh </programlisting>
+
+        <para>For more information about Spark Cluster options, see the
+        following pages.</para>
+
+        <itemizedlist>
+          <listitem>
+            <para><ulink
+            url="https://spark.apache.org/docs/latest/spark-standalone.html#cluster-launch-scripts ">https://spark.apache.org/docs/latest/spark-standalone.html#cluster-launch-scripts</ulink></para>
+          </listitem>
+
+          <listitem>
+            <para><ulink
+            url="https://spark.apache.org/docs/latest/configuration.html#environment-variables ">https://spark.apache.org/docs/latest/configuration.html#environment-variables</ulink></para>
+          </listitem>
+        </itemizedlist>
+
+        <sect3 id="SampleSparkConfigurations">
+          <title>Sample Optional Configurations</title>
```

Comment:
Would "Example Uses Cases" be a better title for this section>

Reply:
sure